app/models/pageflow/published_entry.rb

```diff
@@ -31,6 +33,10 @@ def initialize(entry, revision = nil)
       @custom_revision = !!revision
     end
 
+    def title
+      revision.title.presence || entry.title
+    end
+
```

Comment:
This is a good idea. What do you think about also displaying `entry.title` as an grey placeholder inside the title input field in the editor? That would make this logic clear for the editor.